src/ontology/edits/oeo-physical.omn

```diff
@@ -2679,22 +2738,21 @@ Class: SolarPhotovoltaic
 Class: SolarPowerPlant
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Solar power is the conversion of energy from sunlight into electricity, either directly using photovoltaics (PV), indirectly using concentrated solar power, or a combination. Concentrated solar power systems use lenses or mirrors and tracking systems to focus a large area of sunlight into a small beam. Photovoltaic cells convert light into an electric current using the photovoltaic effect."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000119> "https://en.wikipedia.org/w/index.php?title=Solar_power&oldid=904827092"@en
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A SolarPowerplant is a Powerplant having an aggregate of SolarPowerUnits as its PowerGeneratingUnits."@en
     
     SubClassOf: 
-        PowerGeneratingUnit,
-        Turbine or (<http://purl.obolibrary.org/obo/BFO_0000051> some PVPanel)
+        RegenerativePowerplant,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some PVPanel
     
     
 Class: SolarThermalCollector
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A solar thermal collector collects heat by absorbing sunlight. The term \"solar collector\" commonly refers to a device for solar hot water heating, but may refer to large power generating installations such as solar parabolic troughs and solar towers or non water heating devices such as solar air heaters."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A solar thermal collector is a heater that absorbs solar radiation to convert it into heat."@en,
         <http://purl.obolibrary.org/obo/IAO_0000119> "https://en.wikipedia.org/w/index.php?title=Solar_thermal_collector&oldid=903948846"@en
     
     SubClassOf: 
-        EnergyGenerator,
+        Heater,
         has_physical_output some Power,
```

Comment:
I think `has_physical_output some Power` does not make sense here. It might be `has_physical_output some heat` would be correct, but this is something that the `SolarThermalCollector` inherit from `Heater`.

Reply:
deleted